src/Routing.re

```diff
@@ -13,33 +13,40 @@ let make = () => {
     <MaterialUi.Container maxWidth=`Sm>
       <CreateCommunityPage />
     </MaterialUi.Container>
-  | [communityName] =>
-    <MaterialUi.Container maxWidth=`Sm>
-      <CommunityStartPage communityName />
-    </MaterialUi.Container>
   | [communityName, "admin", ...subRoute] =>
     <MaterialUi.Container maxWidth=`Lg>
-      <CommunityAdminPage communityName subRoute />
-    </MaterialUi.Container>
-  | [communityName, "top"] =>
-    <MaterialUi.Container maxWidth=`Lg>
-      <TopBoard communityName />
-    </MaterialUi.Container>
-  | [communityName, "history"] =>
-    <MaterialUi.Container maxWidth=`Sm>
-      <ResultHistory communityName />
-    </MaterialUi.Container>
-  | [communityName, playerName] =>
-    <MaterialUi.Container maxWidth=`Sm>
-      <PlayerResults communityName playerName />
-    </MaterialUi.Container>
-  | [communityName, player1Name, player2Name] =>
-    <MaterialUi.Container maxWidth=`Sm>
-      <HeadToHeadPage communityName player1Name player2Name />
-    </MaterialUi.Container>
-  | _ =>
-    <MaterialUi.Container maxWidth=`Sm>
-      <div> {text("Invalid route")} </div>
-    </MaterialUi.Container>
+      <CommunityAdminLoginPage communityName subRoute />
+    </MaterialUi.Container>
+  | [communityName, ...subRoute] =>
+    switch (subRoute) {
+    | [] =>
+      <MaterialUi.Container maxWidth=`Sm>
+        <CommunityStartPage communityName />
+      </MaterialUi.Container>
+    // TODO: Extract TopBoard to top level, since it is going to be subscription-based anyway?
+    | ["top"] =>
+      <MaterialUi.Container maxWidth=`Lg>
+        <div> {text("PLACEHOLDER FOR TOPBOARD: " ++ communityName)} </div>
+      </MaterialUi.Container>
+    // <TopBoard communityName />
+    // TODO: Parametrize the history route
```

Comment:
Remove comments. This is going to be implemented in https://github.com/Yakimych/relogify/issues/80